### PR TITLE
Chore: Bump lezer

### PR DESCRIPTION
**Pull request by zoltanbedi**
Please take care of releasing this if you find it useful.

### CODE REVIEW

**Review by ivanahuckova — APPROVED** [review]

File: tools/tree-viz.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
     <script type="importmap">
       {
         "imports": {
-          "@grafana/lezer-logql": "https://unpkg.com/@grafana/lezer-logql@0.0.11/index.es.js",
-          "@lezer/lr": "https://unpkg.com/@lezer/lr@0.15.8/dist/index.js",
-          "@lezer/common": "https://unpkg.com/@lezer/common@0.15.11/dist/index.js"
+          "@grafana/lezer-logql": "https://unpkg.com/@grafana/lezer-logql@0.0.13/index.es.js",
+          "@lezer/lr": "https://unpkg.com/@lezer/lr@1.0.0/dist/index.js",
```
Comment (ivanahuckova) [inline tools/tree-viz.html:12]: Actually, this `@lezer/lr": "https://unpkg.com/@lezer/lr@1.0.0/dist/index.js` specific change breaks the tree viz tool. It gives `module import failed. this requires a browser with import map support.`. The issue is present from `@lezer/lr@16.0.0` and `15.8` is the last working version for this. 
image

**Review by ivanahuckova — COMMENTED** [review]
Actually, I've found out that tree viz is not working with this change. +cc @gabor

**Comment by gabor-grafana** [comment]
@ivanahuckova hmm.. i'll look into this problem, but i think for now we can merge this, we can fix the `tree-viz.html` problem in a separate pull request somehow.

**Comment by gabor** [comment]
@ivanahuckova hmm.. i'll look into this problem, but i think for now we can merge this, we can fix the `tree-viz.html` problem in a separate pull request somehow.

**Pull request merged**